index.js

```diff
@@ -108,6 +127,31 @@ function outputChangelog (argv, cb) {
   })
 }
 
+function getNextVersion (release) {
+  var newVersion = semver.inc(pkg.version, release.releaseAs)
+
+  if (argv.preRelease) {
```

Comment:
hmm.. I think you could just do something like `semver.inc(pkg.version, 'premajor', argv.tag)` if it's a major bump.